installer/pkg/common/storage.go

```diff
@@ -87,6 +89,11 @@ func StorageConfig(context *RenderContext) storageconfig.StorageConfig {
 	// 5 GiB
 	res.BlobQuota = 5 * 1024 * 1024 * 1024
 
+	_ = context.WithExperimental(func(ucfg *experimental.Config) error {
+		res.Stage = storageconfig.Stage(ucfg.Workspace.Stage)
```

Comment:
Just a thought - if the `Stage` isn't added in the config file, this gives a null pointer error. Do we want to add a check in to ensure that doesn't happen?

I'll approve it and you can merge if you're happy to proceed without it